static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -56,10 +56,10 @@ export function getCoverageColors(
   lines: [number, string][],
   lineCov: LineCoverage[]
 ): Array<Color | 'transparent'> {
-  const lineCoverage = keyBy(lineCov, 'lineNo');
```

Comment:
Quick fix for now - I'll look into this more as we test the feature. It would be nice to still be able to access the coverage with `.lineNo` and `.coverage`.